tools/tsp-client/package.json

```
@@ -34,7 +34,8 @@
     "prettier": "^3.0.1",
     "rimraf": "^5.0.1",
     "ts-node": "^10.9.1",
-    "typescript": "^5.1.6"
+    "typescript": "^5.1.6",
+    "@typespec/compiler": "0.48.1"
```

```suggestion
    "@typespec/compiler": "0.50.0"
```

Probably want to use latest here?

---

updated